Makefile

```diff
@@ -136,6 +127,10 @@ endef
 
 .PHONY: test-deps
 test-deps: node_modules
+	-mkdir -p test-logs
+	-rm logic-inspector/test-logs
+	-ln -s '../test-logs' logic-inspector/test-logs
+	-cd logic-inspector; make; cd ..
```

Comment:
no change needed, but fyi: the "cd .." has no effect since each line runs in its own shell.  Also, this could alternately be "-make -C logic-inspector", or better yet "-$(MAKE) -C logic-inspector" if one aspired to make fanciness and wanted to force those looking at this file to become fancier too ;).